bookshelf.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+class Book:
+    def __init__(self, title, author, pages, price, quantity, sales):
+        self.title = title
+        self.author = author
+        self.pages = pages
+        self.price = price
+        self.quantity = quantity
+        self.sales = sales
+
+    def __str__(self):
+        return f"{self.title} by {self.author}"
+
+    def __repr__(self):
+        return f"Book({self.title}, {self.author}, {self.price}, {self.pages}, {self.quantity}, {self.sales})"
+
+
+class BookShop:
+    def __init__(self):
+        self.books = []
+
+    def add_book(self, book):
+        self.books.append(book)
+
+    def del_book(self, book):
+        self.books.remove(book)
+
+    def top_books_by_price(self, n):
```

Comment:
method should contain verbs
 like: 'def get_top_books_by_price(self, n):'

Reply:
Renamed them